.claude/skills/coverage-test-picker/SKILL.md

```
@@ -0,0 +1,265 @@
+### If both available
+
+1. Create worktree: `wt-new ComfyUI_frontend test/<branch-name>`
+2. Move task file into the worktree: `mv temp/plans/test-task-*.md <worktree-path>/temp/plans/`
+3. Spawn tmux window: `tmux new-window -c <worktree-path> -n "test-<filename>"`
+4. Done — tell user "Task file ready in new tmux window."
+
+### If not available
+
+Print what's missing and how to install:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)